Sources/Commands/Options.swift

```diff
@@ -13,57 +13,18 @@ import POSIX
 import PackageModel
 import Utility
 
-// FIXME: The functionality to find the package root path etc should move to `SwiftTool`, and `Options` should be a struct that gets initialized in a clean way other than just on demand.  Right now, for example, if you happen to ask for `path.root` (which doesn't mean the path of the root directory but rather the top-level package directory path) before you change directory to the path specified by `chdir`, you get the wrong value.  This doesn't need to be so complicated, and in particular, behavior shouldn't depend on the order in which properties are asked for.
 public class Options {
     public var chdir: AbsolutePath?
-    public var path = Path()
     public var enableNewResolver = false
     public var colorMode: ColorWrap.Mode = .Auto
     public var verbosity: Int = 0
     public var buildPath: AbsolutePath?
 
-    public class Path {
```

Comment:
This is awesome!  This has been annoying me since I saw it, and I've had an op-fix note to go back and yank it.  Thanks a lot for this; really good cleanup.